src/Teams/Domain/ProgramTextModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using Teams.Domain;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Teams.Models
+{
+    [Table("Queue")]
+    public class ProgramTextModel
+    {
+        [Key]
```

Comment:
Let's use Fluid API instead of using attributes https://docs.microsoft.com/en-us/ef/core/modeling/
It's better not to couple tightly the domain model and the EF, if it's possible.